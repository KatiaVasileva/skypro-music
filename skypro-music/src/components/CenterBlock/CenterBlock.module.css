.main {
    width: auto;
    -webkit-box-flex: 3;
    -ms-flex-positive: 3;
    flex-grow: 3;
    padding: 20px 40px 20px 111px;
}

.search {
    width: 100%;
    border-bottom: 1px solid #4e4e4e;
    margin-bottom: 51px;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-orient: horizontal;
    -webkit-box-direction: normal;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
}

.searchSvg {
    width: 17px;
    height: 17px;
    margin-right: 5px;
    stroke: #ffffff;
    fill: transparent;
    margin-bottom: 14px;


    &:hover {
        fill: transparent;
        stroke: #acacac;
        cursor: pointer;
    }

    &:active {
        fill: transparent;
        stroke: #ffffff;
        cursor: pointer;
    }
}

.searchText {
    -webkit-box-flex: 100;
    -ms-flex-positive: 100;
    flex-grow: 100;
    background-color: transparent;
    border: none;
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    color: #ffffff;
    margin-bottom: 14px;

    &::placeholder {
        color: #ffffff;
        font-style: normal;
        font-weight: 400;
        font-size: 16px;
        line-height: 24px;
    }
}

.title {
    font-style: normal;
    font-weight: 400;
    font-size: 64px;
    line-height: 72px;
    letter-spacing: -0.8px;
    margin-bottom: 45px;
}

.content {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-orient: vertical;
    -webkit-box-direction: normal;
    -ms-flex-direction: column;
    flex-direction: column;
}

.playlistContent {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-orient: vertical;
    -webkit-box-direction: normal;
    -ms-flex-direction: column;
    flex-direction: column;
    overflow-y: auto;
}

.playlistItem {
    width: 100%;
    display: block;
    margin-bottom: 12px;
}

.playlistTrack {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-orient: horizontal;
    -webkit-box-direction: normal;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-pack: justify;
    -ms-flex-pack: justify;
    justify-content: space-between;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
}

.trackTitle {
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-orient: horizontal;
    -webkit-box-direction: normal;
    -ms-flex-direction: row;
    flex-direction: row;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    width: 447px;
}

.imageContainer {
    position: relative;
}

.trackTitleImage {
    width: 51px;
    height: 51px;
    padding: 16px;
    background: #313131;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -ms-flex-pack: center;
    justify-content: center;
    margin-right: 17px;
}

.trackTitleSvg {
    width: 18px;
    height: 17px;
    fill: transparent;
    stroke: #4e4e4e;
}

@keyframes bubble_out {

    0%,
    to {
        transform: scale(0.5);
    }

    50% {
        transform: scale(1);
    }
}

.playingDot, .playingDotAnimated {
    position: absolute;
    width: 16px;
    height: 16px;
    margin-top: 18px;
    margin-left: 18px;
    background-color: #b672ff;
    border-radius: 8px;
    display: block;
    z-index: 2;
}

.playingDotAnimated {
    animation: bubble_out 0.6s ease-in-out infinite both;
}

.playingDotInactive {
    display: none;
}

.trackTitleLink {
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    color: #ffffff;
}

.trackTitleSpan {
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    color: #4e4e4e;
}

.trackAuthor {
    width: 321px;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: start;
    -ms-flex-pack: start;
    justify-content: flex-start;
}

.trackAuthorLink {
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    color: #ffffff;
    text-align: left;
}

.trackAlbum {
    width: 245px;
}

.trackAlbumLink {
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    color: #696969;
}

.trackTimeSvg {
    width: 14px;
    height: 12px;
    margin-right: 17px;
    fill: transparent;
    stroke: #696969;
}

.trackTimeText {
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 24px;
    text-align: right;
    color: #696969;
}
