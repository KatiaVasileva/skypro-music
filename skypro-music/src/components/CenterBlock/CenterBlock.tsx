import styles from "./CenterBlock.module.css";

function CenterBlock() {
  const classNames = require("classnames");

  return (
    <div className={styles.main}>
      <div className={styles.search}>
        <svg className={styles.searchSvg}>
          <use xlinkHref="/img/icon/sprite.svg#icon-search" />
        </svg>
        <input
          className={styles.searchText}
          type="search"
          placeholder="Поиск"
          name="search"
        />
      </div>
      <h2 className={styles.title}>Треки</h2>
      <div className={styles.filter}>
        <div className={styles.filterTitle}>Искать по:</div>
        <div className={styles.filterButton}>исполнителю</div>
        <div className={styles.filterButton}>году выпуска</div>
        <div className={styles.filterButton}>жанру</div>
      </div>
      <div className={styles.content}>
        <div className={styles.contentTitle}>
          <div
            className={classNames(
              styles.playlistColumn,
              styles.playlistColumn01
            )}
          >
            Трек
          </div>
          <div
            className={classNames(
              styles.playlistColumn,
              styles.playlistColumn02
            )}
          >
            Исполнитель
          </div>
          <div
            className={classNames(
              styles.playlistColumn,
              styles.playlistColumn03
            )}
          >
            Альбом
          </div>
          <div
            className={classNames(
              styles.playlistColumn,
              styles.playlistColumn04
            )}
          >
            <svg className={styles.playlistTitleSvg}>
              <use xlinkHref="/img/icon/sprite.svg#icon-watch"></use>
            </svg>
          </div>
        </div>
        <div className={styles.playlistContent}>
          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Guilt <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Nero
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Welcome Reality
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>4:44</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Elektro <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Dynoro, Outwork, Mr. Gee
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Elektro
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>2:22</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    I’m Fire <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Ali Bakgor
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  I’m Fire
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>2:22</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Non Stop
                    <span className={styles.trackTitleSpan}>(Remix)</span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Стоункат, Psychopath
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Non Stop
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>4:12</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Run Run
                    <span className={styles.trackTitleSpan}>(feat. AR/CO)</span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Jaded, Will Clarke, AR/CO
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Run Run
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>2:54</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Eyes on Fire
                    <span className={styles.trackTitleSpan}>
                      (Zeds Dead Remix)
                    </span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Blue Foundation, Zeds Dead
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Eyes on Fire
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>5:20</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Mucho Bien
                    <span className={styles.trackTitleSpan}>
                      (Hi Profile Remix)
                    </span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  HYBIT, Mr. Black, Offer Nissim, Hi Profile
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Mucho Bien
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>3:41</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>
          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>
          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>
          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>
          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>
          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Knives n Cherries
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  minthaze
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Captivating
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>1:48</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    How Deep Is Your Love
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Calvin Harris, Disciples
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  How Deep Is Your Love
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>3:32</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    Morena <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://">
                  Tom Boxer
                </a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://">
                  Soundz Made in Romania
                </a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}>3:36</span>
              </div>
            </div>
          </div>

          <div className={styles.playlistItem}>
            <div className={styles.playlistTrack}>
              <div className={styles.trackTitle}>
                <div className={styles.trackTitleImage}>
                  <svg className={styles.trackTitleSvg}>
                    <use xlinkHref="/img/icon/sprite.svg#icon-note"></use>
                  </svg>
                </div>
                <div>
                  <a className={styles.trackTitleLink} href="http://">
                    <span className={styles.trackTitleSpan}></span>
                  </a>
                </div>
              </div>
              <div className={styles.trackAuthor}>
                <a className={styles.trackAuthorLink} href="http://"></a>
              </div>
              <div className={styles.trackAlbum}>
                <a className={styles.trackAlbumLink} href="http://"></a>
              </div>
              <div>
                <svg className={styles.trackTimeSvg}>
                  <use xlinkHref="/img/icon/sprite.svg#icon-like"></use>
                </svg>
                <span className={styles.trackTimeText}></span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}

export default CenterBlock;
