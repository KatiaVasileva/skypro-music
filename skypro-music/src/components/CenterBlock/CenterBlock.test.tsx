import ReduxProvider from "../../store/ReduxProvider";
import CenterBlock from "./CenterBlock";
import { render } from "@testing-library/react";

describe("TrackComponent", () => {
  it("renders correctly", () => {
    const { asFragment } = render(
        <ReduxProvider>
        <CenterBlock allTracks={[]} />
      </ReduxProvider>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});