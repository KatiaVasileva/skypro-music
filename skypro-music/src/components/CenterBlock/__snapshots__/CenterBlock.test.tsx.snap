// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackComponent renders correctly 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <div
      class="search"
    >
      <div
        class=""
      >
        <svg
          class="searchSvg"
        >
          <use
            xlink:href="/img/icon/sprite.svg#icon-search"
          />
        </svg>
      </div>
      <input
        class="searchText"
        name="search"
        placeholder="Поиск"
        type="search"
      />
    </div>
    <h2
      class="title"
    >
      Треки
    </h2>
    <div
      class="filter"
    >
      <div
        class="filterTitle"
      >
        Искать по:
      </div>
      <div
        class="filterButton"
      >
        исполнителю
      </div>
      <div
        class="filterButton"
      >
        году выпуска
      </div>
      <div
        class="filterButton"
      >
        жанру
      </div>
    </div>
    <div
      class="content"
    >
      <div
        class="contentTitle"
      >
        <div
          class="playlistColumn playlistColumn01"
        >
          Трек
        </div>
        <div
          class="playlistColumn playlistColumn02"
        >
          Исполнитель
        </div>
        <div
          class="playlistColumn playlistColumn03"
        >
          Альбом
        </div>
        <div
          class="playlistColumn playlistColumn04"
        >
          <div
            class=""
          >
            <svg
              class="playlistTitleSvg"
            >
              <use
                xlink:href="/img/icon/sprite.svg#icon-watch"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        class="playlistContent"
      />
    </div>
  </div>
</DocumentFragment>
`;
